src/services/repository.ts

```diff
@@ -120,13 +129,38 @@ export default class Repository {
 
   async #fetchOpenSSFScorecard(): Promise<scorecard.ScorecardResult | null> {
     try {
-      return await scorecard.result(`${this.org.orgName}/${this.repository.name}`);
+      return await scorecard.result(`${this.#org.orgName}/${this.#repository.name}`);
     }
     catch {
       return null;
     }
   }
 
+  async #lastRelease(repository: string, version: string): Promise<string | null> {
+    try {
+      const { stdout } = await asyncExec(`npm view ${repository} time'[${version}]'`);
+
+      const date = new Date(stdout.split(EOL)[0].trim());
+
+      return kDateFormatter.format(date);
+    }
+    catch {
+      return null;
+    }
+  }
```

Comment:
Hmm maybe there is a better way using: https://github.com/NodeSecure/npm-registry-sdk

like packument